src/pages/home.html

```diff
@@ -0,0 +1,1926 @@
+                            قاب عکس شیب دار
+                        </p>
+                        <p class="mt-3 text-xl font-bold text-bondi-blue">
+                            ۲۴٬۰۰۰تومان- ۶۸٬۰۰۰ تومان
+                        </p>
+                    </a>
+                    <img
+                        src="/src/assets/images/home/circle-with-lines.png"
+                        alt="عکس دایره با خطوط"
+                        class="absolute -top-[17%] right-[53%] -z-10"
+                    />
+                </div>
+            </section>
+            <!-- greeting section -->
+
+            <!-- recommended plans -->
+            <section class="mt-20 bg-link-water">
+                <div class="relative mx-auto mb-20 max-w-7xl py-24">
+                    <img
+                        src="/src/assets/images/bookmark.png"
+                        alt=""
+                        class="absolute -left-[3%] -top-[3%]"
+                    />
+                    <div class="relative flex items-center justify-center">
+                        <img
+                            src="/src/assets/images/home/shape-curly3.png"
+                            alt=""
+                            class="absolute -top-[80%] right-[25%]"
+                        />
+                        <h3 class="text-5xl text-black">
+                            <span class="text-amaranth">پلن های </span>پیشنهادی
+                            عکسخونه
+                        </h3>
+                    </div>
+                    <div class="mt-8 flex items-center justify-center gap-2">
+                        <img
+                            src="/src/assets/images/light-shape.png"
+                            alt=""
+                            class="rotate-180"
+                        />
+                        <p class="text-lg tracking-wider text-bali-hai">
+                            بهترین انتخاب در کمترین زمان ممکن
+                        </p>
+                        <img
+                            src="/src/assets/images/light-shape.png"
+                            alt=""
+                            class=""
+                        />
+                    </div>
+                    <div
+                        class="relative mt-24 grid grid-cols-3 items-center justify-center gap-8"
+                    >
+                        <img
+                            src="/src/assets/images/home/plant-shape.png"
+                            alt=""
+                            class="absolute -right-[5%] -top-[16%]"
+                        />
+                        <div
+                            class="flex flex-col transition-all duration-200 ease-linear hover:-translate-y-8 hover:shadow-hover"
+                        >
+                            <div
+                                class="relative flex h-[233px] w-[418px] items-center justify-center overflow-hidden bg-west-side"
```

Comment:
Using fixed width inside grid item causes this:
The children are wider than the parent. Because the parent width is dynamic (1/3 with grid) but the child width is fixed!
(The white transparent box shown in image is parent size)
![image](https://github.com/TechNullogy/Simple-Shop/assets/17345129/18c7623e-4afb-4db6-b58d-3d9c107512a9)

Reply:
I fixed that.but I am be curious why this should be a problem at all?

Reply:
It's important because we should prefer automatic size & position according to responsive design. Otherwise, when implementing the page for other resolutions we need to set the element size for every resolution (break-point) and it's painful!